Web/zendesk.2m.rb

```diff
@@ -0,0 +1,63 @@
+#!/usr/bin/env ruby
+
+# <xbar.title>Zendesk</xbar.title>
+# <xbar.author>Matt Roberts, James Stiff for the icons</xbar.author>
+# <xbar.author.github>mattwoberts</xbar.author.github>
+# <xbar.image>https://i.imgur.com/e6rdUBh.png</xbar.image>
+# <xbar.dependencies>Ruby</xbar.dependencies>
+# <xbar.desc>Shows non-solved Zendesk tickets count</xbar.desc>
+
+# Configurations
+ZENDESK_SUBDOMAIN = ""
```

Comment:
@mattwoberts this is great, but I wonder if you could use the [vars](https://xbarapp.com/docs/2021/03/14/variables-in-xbar.html)?